Modules/_ssl.c

```diff
@@ -4427,6 +4427,12 @@ _ssl__SSLContext_load_dh_params_impl(PySSLContext *self, PyObject *filepath)
     FILE *f;
     DH *dh;
 
+#if defined(MS_WINDOWS) && defined(_DEBUG)
```

Comment:
Is `_DEBUG` the canonical "NDEBUG" for Windows builds? (just asking)

Reply:
Ah yes, what I meant is "is it common to use the _DEBUG macro instead of the NDEBUG one" (I think I've only seen Py_DEBUG and NDEBUG usage but not _DEBUG)

Reply:
I guess it's personal preference? `_DEBUG` is [defined by the compiler based on the runtime library](https://learn.microsoft.com/en-us/cpp/c-runtime-library/debug?view=msvc-170) which is what I wanted, while `NDEBUG` [controls the behaviour of `assert`](https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/assert-macro-assert-wassert?view=msvc-170).

So it's technically possible to define them both at once, but generally you don't, and I wouldn't suggest it in a codebase like ours because it's almost certainly assuming they are mutually exclusive.